wger/nutrition/models.py

```diff
@@ -512,6 +512,17 @@ def get_owner_object(self):
         """
         return False
 
+    def _get_energy_kilojoule(self):
+        """
+        returns kilojoules for current ingredient, 0 if energy is uninitialized
+        """
+        if self.energy:
+            return self.energy * 4184
```

Comment:
Sorry, 1kcal ist 4.1 kJ, not four thousand! :D I had just copy pasted it from the German wikipedia and hadn't thought that not everybody uses commas for this